JPS_Ontology/ontology/ontosensorloggermobileapp/OntoSensorLoggerMobileApp.csv

```diff
@@ -0,0 +1,53 @@
+Source,Type,Target,Relation,Domain,Range,Quantifier,Comment,Defined By,Label
+OntoSensorLoggerMobileApp,TBox,http://www.theworldavatar.com/ontology/ontosensorloggermobileapp/ontoSensorLoggerMobileApp.owl,https://www.w3.org/2007/05/powder-s#hasIRI,,,,,,
+OntoSensorLoggerMobileApp,TBox,1,http://www.w3.org/2002/07/owl#versionInfo,,,,,,
+OntoSensorLoggerMobileApp,TBox,The OntoSensorLoggerMobileApp ontology (OntoSensorLoggerMobileApp) is an ontology developed to represent the detailed information and spatial relationships,http://www.w3.org/2000/01/rdf-schema#comment,,,,,,
+OntoSensorLoggerMobileApp,TBox,,http://www.w3.org/2002/07/owl#imports,,,,,,
+http://www.ontology-of-units-of-measure.org/resource/om-2/Measure,Class,,,,,,,http://www.ontology-of-units-of-measure.org/resource/om-2/,Measure
```

Comment:
There are several definitions here of things that are defined elsewhere. These are redundant - please remove them.